src/main/java/org/jenkinsci/plugins/workflow/cps/steps/ParallelStep.java

```diff
@@ -46,7 +46,7 @@
     /**
      * All the sub-workflows as {@link Closure}s, keyed by their names.
      */
-    /*package*/ final Map<String,Closure> closures;
+    /*package*/ final transient Map<String,Closure> closures;
```

Comment:
Why? `ParallelStep` was not even `Serializable`, nor should anything be saving it via XStream, so this makes no sense.